src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -283,7 +340,14 @@ def get_counter(self, sub_id, table_name):
             counter_value = 0
             for lag_member in self.lag_name_if_name_map[self.oid_lag_name_map[oid]]:
                 counter_value += self._get_counter(mibs.get_index(lag_member), table_name)
-
+            # import pdb; pdb.set_trace()
+            sai_lag_id = self.lag_sai_map[self.oid_lag_name_map[oid]]
+            sai_lag_rif_id = self.port_rif_map[sai_lag_id]
+            if sai_lag_rif_id in self.rif_port_map:
+                _table_name = bytes(getattr(table_name, 'name', table_name), 'utf-8')
```

Comment:
>_table_name [](start = 16, length = 11)

leading underscore is for private class member. Could you use other name convention?

Reply:
Sure